server/base/src/main/java/org/apache/accumulo/server/conf/ServerConfigurationFactory.java

```diff
@@ -51,9 +65,18 @@ public class ServerConfigurationFactory extends ServerConfiguration {
   private final SiteConfiguration siteConfig;
   private final DeleteWatcher deleteWatcher = new DeleteWatcher();
 
+  private static final int REFRESH_PERIOD_MINUTES = 15;
```

Comment:
The watcher should fire and update when the change is written to ZooKeeper - this is just to ensure that if a notification was missed this will make it (eventually) consistent.  Ideally, this would never find anything that is not up to date.

Reply:
ok, great. Thanks.